Modelica/Electrical/Polyphase/Sensors/AronSensor.mo

```diff
@@ -69,17 +69,19 @@ equation
           extent={{-30,-10},{30,-70}},
           textColor={64,64,64},
           textString="W")}),      Documentation(info="<html>
-<p>
-Contains two 
-<a href=\"modelica://Modelica.Electrical.Analog.Sensors.PowerSensor\">power meters</a>
-to measure total active power in a three-phase system.
-</p>
-<p>
-This device works only for sinusoidal voltages and currents in three-phase systems without neutral.
-</p>
-<p>
-Elmar Schr&uuml;fer, Leonhard Reindl, Bernhard Zagar: Elektrische Messtechnik. Carl Hanser Verlag.<br>
-Reinhard Lerch: Elektrische Messtechnik. Springer Vieweg.
-</p>
+<p>Contains two <a href=\"modelica://Modelica.Electrical.Analog.Sensors.PowerSensor\">power meters</a> to measure total active power in a three-phase system. </span></p>
```

Comment:
The closing `</span>` tags shall be removed in this and all other lines.

Reply:
Sorry for this leftover.
I think now the file is ok.